src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts

```diff
@@ -147,24 +147,26 @@ export class HealthDeclarationCatADIPart2Page extends BasePageComponent {
         select(getCandidateDriverNumber),
         map(formatDriverNumber),
       ),
-      passCertificateNumber$: currentTest$.pipe(
-        select(getPassCompletion),
-        select(getPassCertificateNumber),
-      ),
-      licenseProvided$: currentTest$.pipe(
-        select(getPassCompletion),
-        map(isProvisionalLicenseProvided),
-      ),
+      // passCertificateNumber$: currentTest$.pipe(
+      //   select(getPassCompletion),
+      //   select(getPassCertificateNumber),
+      // ),
+      // licenseProvided$: currentTest$.pipe(
+      //   select(getPassCompletion),
+      //   map(isProvisionalLicenseProvided),
+      // ),
```

Comment:
same here

Reply:
will be removed after i rebase and your changes are in!